search/search-dev-tools/src/components/query/index.js

```diff
@@ -35,19 +34,19 @@ const Query = ( { args, request, url, query_args, backtrace = [] } ) => {
 	const queryResultRef = useRef( null );
 
 	/**
-	 * 
+	 *
 	 * @param {Object} query the query to Run
-	 * @param {String} url ES url
+	 * @param {String} _url ES url
 	 */
-	const fetchForQuery = async ( query, url ) => {
+	const fetchForQuery = async ( query, _url ) => {
```

Comment:
What if we rename it to `es_url` or something like that? An underscore prefix might have some added meaning that we don't want

Reply:
I'm not even sure it's a good rule in this context TBH. We're not mutating the `url` nor we re-declaring it. My vote would be to skip this change, and instead slap the `eslint-disable-line no-use-before-define no-shadow` on both occurrences .

Additionally, it will break the useEffect callback on L64: https://github.com/Automattic/vip-go-mu-plugins/pull/2262/files#diff-a16e28a93d61fa9558c75f898bba81f5c500e9fccdec1e093a29ffa5027eed9dL65

Reply:
I couldn't decide which way to go ... so I ignored you both and just deleted the code :)

(nah, actually as @rinatkhaziev mentioned we are not actually mutating the argument so there is no reason to not use the variable that is being overwritten I think, let me know what you think)

Reply:
Looks cleaner now, I like it.